src/plugins/template/tests/functional/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ ov_add_test_target(
         LINK_LIBRARIES
             openvino::funcSharedTests
             openvino::runtime::dev
+        EXCLUDED_SOURCE_PATHS ${EXCLUDED_SOURCE_PATHS}
```

Comment:
looks like we have no `EXCLUDED_SOURCE_PATHS` defined

Reply:
It should be removed as is not required here.